### ISSUE TITLE
Failed to minify when build

### ISSUE THREAD

**Issue by gravitano** [rakannimer/react-google-charts#224]
Hi, i have problem when builing react app using this plugin. It's say can't minify the `./node_modules/react-google-charts/dist/index.esm.js:27 `.

```
> cross-env REACT_APP_BUILD_ENV=production react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/react-google-charts/dist/index.esm.js:27 

Read more here: http://bit.ly/2tRViJ9
```

**Comment by rakannimer** [comment]
Hey @gravitano,

Thanks for reporting that !

I set the target to es5 and published as `2.0.28`.

It should let you compile successfully now !

Cheers !

**Comment by gravitano** [comment]
Hey @rakannimer ,

I just tested the new version and it work like a charm.

Thank you!
